system-variables.md

```diff
@@ -2825,21 +2842,13 @@ explain select * from t where age=5;
 - Default value: `OFF`
 - This variable is used to control whether to allow `INSERT`, `REPLACE`, and `UPDATE` statements to operate on the `_tidb_rowid` column. This variable can be used only when you import data using TiDB tools.
 
-### tidb_opt_force_inline_cte <span class="version-mark">New in v6.3.0</span>
-
-- Scope: SESSION | GLOBAL
-- Persists to cluster: Yes
-- Type: Boolean
-- Default value: `OFF`
-- This variable is used to control whether common table expressions (CTEs) in the entire session are inlined or not. The default value is `OFF`, which means that inlining CTE is not enforced by default. However, you can still inline CTE by specifying the `MERGE()` hint. If the variable is set to `ON`, all CTEs (except recursive CTE) in this session are forced to be inlined.
-
 ### tidb_optimizer_selectivity_level
 
-- Scope: SESSION | GLOBAL
+- Scope: SESSION
 - Persists to cluster: Yes
-- Default value: `1`
-- Value options: `1` and `2` (not recommended)
-- This variable controls the iteration of the optimizer's estimation logic. After changing the value of this variable, the estimation logic of the optimizer will change greatly. Currently, `1` is the only valid value. It is not recommended to set the value to `2`.
+- Default value: `0`
+- Range: `[0, 2147483647]`
+- This variable is used to control the selectivity estimation level.
```

Comment:
@winoros Would you please provide more information about this variable? The information you have provided in https://github.com/pingcap/docs/pull/9891/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R2523 does not look correct.